README.md

```diff
@@ -154,6 +154,16 @@ A short guide to Test-driven development in golang. free to [read online](https:
 
 The main goal of this book is to help developers avoid common mistakes while at the same time, learning a new programming language through a "hands-on approach". This book provides a good level of detail on "how to do it securely" showing what kind of security problems could arise during development.
 
+### [Go with the Domain: Building Modern Business Software in Go](https://threedots.tech/go-with-the-domain/) *Free*
+
+<a href="https://threedots.tech/go-with-the-domain/"><img src="https://threedots.tech/img/go-with-domain-cover-160-retina.png" width="120px"/></a>
+
+*Go with the Domain* is a book on building Go applications that solve complex problems in an idiomatic way.
+It features techniques like Domain-Driven Design, Clean Architecture, CQRS, and other patterns.
```

Comment:
I'm not sure what CQRS is. Might be worth spelling it out?

Reply:
Good point, thanks. Added it.

Reply:
Thanks